src/tequila/circuit/visualize.py

```diff
@@ -0,0 +1,128 @@
+import importlib.util
+
+import tequila
+from typing import Optional, List
+import matplotlib.pyplot as plt
+import matplotlib.figure as figure
+import networkx
+
+
+def visualize(qubit_hamiltonian: tequila.QubitHamiltonian,
+              circuit: Optional[tequila.QCircuit] = None,
+              connectivity: Optional[List[tuple]] = None,
+              file_path: Optional[str] = None) -> figure.Figure:
+    """
+    Precondition:
+    The maximum number of qubits is 10 at the moment (Feb24, 2023)
+
+    Post condition:
+    The graph of the qubit_hamiltonian is displayed
+    or is stored in file_path
+
+    One thing to note is that if you are using command-line interface,
+    the plot might not be successfully shown
+
+    , so it's better idea to save it as an image.
+    === parameters ===
+    qubit_hamiltonian: A QubitHamiltonian representation of pauli operators
+    circuit: A QCircuit that corresponds to the
+    connectivity: networkx.Graph.edges that show the connectivity of qubits
+    A nested list should be provided for this argument.
+    file_name: str format of file name
```

Comment:
inconsistent naming: file_name but keyword is currently file_path

Reply:
Ok, I have changed it